gui/public/i18n/ru/translation.ftl

```diff
@@ -0,0 +1,542 @@
+onboarding-reset_tutorial = Сбросить туториал
+onboarding-reset_tutorial-description = Эта функция не завершена, просто нажмите продолжить
+
+## Setup start
+onboarding-home = Добро пожаловать в SlimeVR!
+# This cares about multilines and it's centered!!
+onboarding-home-description =
+    Обеспечение полного отслеживания тела
+    для всех
+onboarding-home-start = Давайте все настроим!
+
+## Enter VR part of setup
+onboarding-enter_vr-back = Вернуться к привязке трекеров
+onboarding-enter_vr-title = Время зайти в VR!
+onboarding-enter_vr-description = Наденьте все ваши трекеры и зайдите в VR!
+onboarding-enter_vr-ready = Я готов
+
+## Setup done
+onboarding-done-title = Вы готовы!
+onboarding-done-description = Наслаждайтесь игре!
+onboarding-done-close = Закрыть гид
+
+## Tracker connection setup
+onboarding-connect_tracker-back = Вернитесь к данным Wi-Fi
+onboarding-connect_tracker-title = Подключить трекеры
+onboarding-connect_tracker-description-p0 = Теперь самое интересное - подключение всех трекеров!
+onboarding-connect_tracker-description-p1 = Просто подключите все, что еще не подключены, через USB-порт.
+onboarding-connect_tracker-issue-serial = У меня проблемы с подключением!
+onboarding-connect_tracker-usb = USB Трекер
+onboarding-connect_tracker-connection_status-connecting = Отправить данные Wi_Fi
+onboarding-connect_tracker-connection_status-connected = Подключен к Wi-Fi
+onboarding-connect_tracker-connection_status-error = Невозможно подключиться к Wi-Fi
+onboarding-connect_tracker-connection_status-start_connecting = Ищем трекеры...
+onboarding-connect_tracker-connection_status-handshake = Подключен к трекеру
+# $amount (Number) - Amount of trackers connected (this is a number, but you can use CLDR plural rules for your language)
+# More info on https://www.unicode.org/cldr/cldr-aux/charts/22/supplemental/language_plural_rules.html
+# English in this case only has 2 plural rules, which are "one" and "other",
+# we use 0 in an explicit way because there is no plural rule in english for 0, so we directly say
+# if $amount is 0 then we say "No trackers connected"
+onboarding-connect_tracker-connected_trackers = { $amount ->
+    [0] Нет трекеров
+    [one] { $amount } трекер
+    [few] { $amount } трекера
+    *[other] { $amount } трекеров
+} подключено
+onboarding-connect_tracker-next = Я подключил все трекеры!
+
+## Tracker assignment setup
+onboarding-assign_trackers-back = Вернуться к данным Wi-Fi
+onboarding-assign_trackers-title = Привязать трекеры
+onboarding-assign_trackers-description = Давайте выберем, какой трекер куда идет. Нажмите на место, где вы хотите разместить трекер
+# Look at translation of onboarding-connect_tracker-connected_trackers on how to use plurals
+# $assigned (Number) - Трекеры, которым была назначена часть тела
+# $trackers (Number) - Трекеры, подключенные к серверу
+onboarding-assign_trackers-assigned = { $assigned } из { $trackers ->
+    [one] 1 трекер
```

Comment:
While it's really unlikely to actually occur for most people this should probably be:
```
[one] { $amount } трекер
```
Otherwise `21 trackers` will be translated to `1 трекер`, which is confusing.